### PR TITLE
Decorate a class

### PR DESCRIPTION
**Pull request by ishimwepati**
Hello!

I did the following:

- Created a `Nameable` class and implemented a method.
- Ensured that the `Person` class inherits from `Nameable`.
- Implemented the `correct_name` method in the `Person` class to return the name attribute.
- Prepared a base `Decorator` class that inherits from `Nameable`.
- Created `CapitalizeDecorator` and `TrimmerDecorator` classes, both inheriting from the base `Decorator` class.
- Implemented the `correct_name` method in the `TrimmerDecorator` to limit the output to a maximum of 10 characters.

Your suggestions are most welcome!

### CODE REVIEW

File: class/person.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Person
```
Comment (gemmen29) [inline class/person.rb]: - Kindly follow the project requirement in the [`Turn your Person class to Nameable`](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Turn%20your%20Person,the%20name%20attribute.) section by:
   - [x] Make sure that your Person class inherits from Nameable.
   - [x] Make sure that this class has a method `correct_name` implemented. It should simply return the name attribute.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ishimwepati, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: class/person.rb (hunk 1 above)
Comment (ishimwepati) [reply class/person.rb]: Ohh thanks, let me just push it !

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ishimwepati, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**